website/docs/releases/development-builds.md

```diff
@@ -5,132 +5,157 @@ hide:
 
 # Development builds
 
+<style>
+span.error {
+  font-weight: bold;
+  font-size: 95%;
+  color: red;
+}
+</style>
+
 <script>
 
-function set_build_version(gh_api_artifacts, os_name) {
-    fetch(gh_api_artifacts)
-        .then(response => {
-            if (response.status !== 200)
-                return;
-
-            response.json().then(data => {
-                let changelog = data.artifacts
-                    .find(a => a.name.startsWith("changelog-"));
-
-                if (changelog === undefined)
-                    return;
-
-                let n = changelog.name.length;
-                let version = changelog.name.substring(10, n - 4);
-                let version_el = document.getElementById(os_name + "-build-version");
-                version_el.textContent = version;
-            });
-        })
-        .catch(err => {
-            console.log('Fetch Error :-S', err);
-        });
+// For local testing only: uncomment and replace API_TOKEN with a valid GitHub
+// API token. This is to bypass the low hourly rate limits for unauthenticated
+// API access during testing (only 60 requests per hour).
+//
+// !!! IMPORTANT -- *NEVER* check in your API token into the repo !!!
+//
+let headers = {
+//  "Authorization": "bearer API_TOKEN"
 }
 
-function handle_error(error_message, os_name) {
-  let build_link_tr_el = document.getElementById(os_name + "-build-link");
-  build_link_tr_el.innerHTML = error_message;
+function get_build_link_tr_el(os_name) {
+  return document.getElementById(os_name + "-build-link")
+}
+function get_build_version_el(os_name) {
+  return document.getElementById(os_name + "-build-version")
+}
+function get_build_date_el(os_name) {
+  return document.getElementById(os_name + "-build-date")
+}
 
-  let version_el = document.getElementById(os_name + "-build-version");
-  version_el.textContent = '';
+function set_build_version(gh_api_artifacts, os_name) {
+  fetch(gh_api_artifacts, { method: "GET", headers: headers })
+    .then(response => {
+      if (response.status !== 200)
+        return
+
+      response.json().then(data => {
+        let version_regexp = "dosbox-staging-[a-zA-Z0-9-]*-([0-9]+\.[0-9]+\.[0-9]+.+)"
+        let release = data.artifacts.find(a => a.name.match(version_regexp))
```

Comment:
Just gonna go ahead and trust you on that one :laughing:

Reply:
> Just gonna go ahead and trust you on that one 😆

Yeah, tested it for a good 5 minutes with a variety inputs and it worked fine. 

Regexps are kinda "write-only" 😆